### DISCUSSION TITLE
What does the numbers parameter mean?

### DISCUSSION THREAD

**Discussion by sagrawal8** [ultralytics/yolov5#4088] [Q&A]
Hi, I was trying to implement the Yolov5 model in TF. I don't seem to understand what the parameter 'number' means down below:

 [from, number, module, args]
[-1, 1, Conv, [256, 3, 2]],  # 3-P3/8
[-1, 9, C3, [256]],

This is present in: https://github.com/ultralytics/yolov5/blob/master/models/yolov5s.yaml

**Comment by glenn-jocher** [comment]
@sagrawal8 the number of times a module is repeated. C3 with n=9.

```python
class C3(nn.Module):
    # CSP Bottleneck with 3 convolutions
    def __init__(self, c1, c2, n=1, shortcut=True, g=1, e=0.5):  # ch_in, ch_out, number, shortcut, groups, expansion
```

In any case YOLOv5 models are already implemented as TF2/Keras models in https://github.com/ultralytics/yolov5/pull/1127

**Reply by sagrawal8** [reply]
Thanks Glenn. Love Ultralytics and everything you do.
